src/commands/build.ts

```diff
@@ -124,10 +135,12 @@ export class BuildCommand implements Command {
     const validBuildOptions = new Set(this.args.map(({name}) => name));
     for (const buildOption of Object.keys(options)) {
       if (validBuildOptions.has(buildOption)) {
-        const [prefix, option] = buildOption.split('-', 2);
+        const [prefix, ...rest] = buildOption.split('-');
         if (['css', 'html', 'js'].indexOf(prefix) !== -1) {
+          const option = dashToCamelCase(rest.join('-'));
           (<any>buildOptions)[prefix] = (<any>buildOptions)[prefix] || {};
```

Comment:
I guess this is tricky because we can't represent functions over string literals well in the type system. But this is fragile and will fail silently in all sorts of ways.

Reply:
Yeah, I mentioned this in the PR description. I feel like we should just directly map flags into the config object rather than trying to magic the CLI (especially once we make cool mapped types for the cli lib).